src/connectionController.ts

```diff
@@ -35,11 +37,11 @@ export enum DataServiceEventTypes {
 export default class ConnectionController {
   // This is a map of connection instance ids to their connection model.
   private _connectionConfigs: {
-    [key: string]: any;
+    [key: string]: ConnectionConfigType;
   } = {};
 
-  _currentConnection: any;
-  private _currentConnectionConfig: any;
+  _currentConnection: DataServiceType | null = null;
```

Comment:
What does `null = null` mean?

Reply:
ah just the type can be `DataServiceType` or `null` and then we're setting it to `null`. hah it does look a little funny though... I'll swap the types around just for readability

Reply:
ahah i was reading it wrong! thx for explaining :)